wurst/systems/spells/mage/Metronome.wurst

```diff
@@ -39,13 +39,28 @@ function onCast()
     var abilityId = table.loadInt(9)
 
     if GetRandomInt(0, 2) == 0
+        //Temporary vision required by dummy
+        var vis = createVision(owner, target.getPos(), 300, true)
+        vis.start()
+        doAfter(0.5) ->
+            vis.destr()
         new DummyCaster().castTarget(owner, abilityId, 1, table.loadInt(abilityId), target, pos)
         return
 
     abilityId = table.loadInt(GetRandomInt(0, 8))
+    //Temporary vision required by dummy
+    var vis2 = createVision(owner, target.getPos(), 300, true)
+    vis2.start()
+    doAfter(0.5) ->
+        vis2.destr()
     new DummyCaster().castTarget(owner, abilityId, 1, table.loadInt(abilityId), target, pos)
 
     doPeriodicallyCounted(1.00, GetRandomInt(3, 5)) cb ->
+        //Temporary vision required by dummy
+        var vis3 = createVision(owner, target.getPos(), 300, true)
+        vis3.start()
+        doAfter(0.5) ->
+            vis3.destr()
```

Comment:
Can we give vision for less than 0.5 seconds? That seems like a decent chunk of time.